packages/kbn-management/settings/application/services.tsx

```diff
@@ -25,7 +45,28 @@ export const SettingsApplicationProvider: FC<SettingsApplicationServices> = ({
   children,
   ...services
 }) => {
-  return <FormProvider {...services}>{children}</FormProvider>;
+  // Destructure the services to avoid a type-widening inclusion of unrelated services.
```

Comment:
This can be important.  If the props contain a lot of extra things, (like expanding `CoreStart` into the `props`), we can destructure to ensure only the items we want end up in the context.